jest.config.base.js

```diff
@@ -20,6 +20,7 @@ module.exports = (dir, env = 'jsdom') => {
       '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
         'identity-obj-proxy',
       '\\.(css|less)$': 'identity-obj-proxy',
+      '^graphql-language-service-([^/]+)': `${__dirname}/packages/graphql-language-service/src/$1`,
```

Comment:
This is required for proper resolution of the `graphql-language-service-*` packages since the other one is only valid for `graphql-language-service` and `graphql-language-service-cli` cases